src/python/Utils/Utilities.py

```diff
@@ -224,6 +227,27 @@ def decodeBytesToUnicode(value, errors="strict"):
     return value
 
 
+def encodeToString(value, errors="strict"):
```

Comment:
I think this name is misleading. We are dealing with strings, and we want to encode a unicode string into a bytes strings. I would suggest using `encodeUnicodeToBytes` instead.